lib/generators/hyrax/templates/db/seeds.rb

```diff
@@ -9,7 +9,14 @@
 # To use this file, run the following command in the .internal_test_app:
 #   rails generate hyrax:sample_data
 
+puts "Creating users"
 user = User.create(email: 'foo@example.org', password: 'foobarbaz')
+user = User.create(email: 'mgr1@example.org', password: 'pppppp') # 6*p
+user = User.create(email: 'mgr2@example.org', password: 'pppppp')
+user = User.create(email: 'dep1@example.org', password: 'pppppp')
+user = User.create(email: 'dep2@example.org', password: 'pppppp')
+user = User.create(email: 'vw1@example.org', password: 'pppppp')
+user = User.create(email: 'vw1@example.org', password: 'pppppp')
```

Comment:
The users should all have different variable names, because user foo is intended to be the depositor of everything, and this change is making the final user created to be the depositor.

Also, vw1@example.org is used twice. I assume it was intended to be vw2 for the final user.